Hangman/ViewController.m

```diff
@@ -37,6 +39,10 @@ - (void)setupUI {
     }
     NSLog(@"%@", newMysteryWordLabel);
     _mysteryWordLabel.text = newMysteryWordLabel;
+    
+    // Guess Counter - set to 10
+    self.guessCounter = 10;
```

Comment:
Any reason why you roll with `self.guessCounter` instead of `_guestCounter`? I don't think it really makes a difference either way since they do the same thing but I was just curious if there was a specific reason for it.